spec.html

```diff
@@ -4253,6 +4253,17 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-isstringprefix" aoid="IsStringPrefix">
+      <h1>IsStringPrefix ( _p_, _q_ )</h1>
```

Comment:
I wonder if this abstract operation could be used in other places too, like `.includes` or `.startsWith`?